### PR TITLE
chore: Migrate all the test typegraphs to the new Python SDK

### PR DESCRIPTION
**Pull request by Natoandro**

### CODE REVIEW

File: typegate/tests/planner/__snapshots__/planner_test.ts.snap [hunk 1]
```diff
@@ -177,97 +177,89 @@ snapshot[`planner 2`] = `
...
 ]
 `;
 
 snapshot[`planner: dependencies 6`] = `
 [
   "two  two  two  object  object_64  false",
-  "two.id  id  two/id  integer  integer_48  false",
-  "two.email  email  two/email  string  UserEmail  true",
-  "two.profile  profile  two/profile  object  object_56  true",
-  "two.profile.email  email  two/profile/email  string  string_52  false",
```
Comment (zifeo) [inline typegate/tests/planner/__snapshots__/planner_test.ts.snap:266]: does not the removal sounds suspicious?

File: typegate/tests/introspection/__snapshots__/union_either_test.ts.snap [hunk 2]
```diff
@@ -46,93 +46,10 @@ snapshot[`Basic introspection 1`] = `
...
-          },
-          {
-            name: "name",
-          },
-        ],
-        kind: "OBJECT",
-        name: "UserOutput",
-        possibleTypes: null,
-      },
-      {
```
Comment (zifeo) [inline typegate/tests/introspection/__snapshots__/union_either_test.ts.snap:62]: does not the removal sounds suspicious?

**Review by zifeo — COMMENTED** [review]
Can you include the typegraph of the current documentation?

I believe we can drop this kind of usage (from the failing tests), there is no usage anymore:
CleanShot 2023-09-24 at 16 51 12@2x

File: typegate/tests/auth/auth_test.ts [hunk 3]
```diff
@@ -177,7 +177,7 @@ Meta.test("Auth", async (t) => {
     const { token } = JSON.parse(await decrypt(cook!));
     const claims = await verifyJWT(token) as JWTClaims;
     assertEquals(claims.accessToken, accessToken);
-    assertEquals(claims.profile?.id, `${id}`);
+    assertEquals(claims.profile?.id, id);
```
Comment (zifeo) [inline typegate/tests/auth/auth_test.ts:180]: this should fail as the profile.id returns a string

File: typegate/tests/introspection/__snapshots__/union_either_test.ts.snap (hunk 2 above)
Comment (zifeo) [reply typegate/tests/introspection/__snapshots__/union_either_test.ts.snap:62]: still unknown?

File: typegate/tests/introspection/__snapshots__/introspection_test.ts.snap [hunk 4]
```diff
@@ -99,36 +99,6 @@ Policies:
         name: "Query",
         possibleTypes: null,
       },
-      {
-        description: "res type",
-        enumValues: null,
-        fields: [
-          {
```
Comment (zifeo) [inline typegate/tests/introspection/__snapshots__/introspection_test.ts.snap:106]: any idea why this drops?

File: typegate/tests/planner/planner.py [hunk 5]
```diff
@@ -97,32 +110,31 @@ def gen_union2():
...
-                                "profile": t.struct(
-                                    {"email": t.email(), "profilePic": t.string()}
-                                ).from_parent("UserProfile")
-                            }
-                        ),
-                        t.string(),
-                        dummy_mat,
-                    ),
+                        code=dummy_func,
+                    ),  # .named("UserProfile"),
```
Comment (zifeo) [inline typegate/tests/planner/planner.py:114]: rm or todo?

File: typegate/tests/simple/__snapshots__/class_syntax_test.ts.snap [hunk 6]
```diff
@@ -51,122 +51,15 @@ snapshot[`Class Syntax 1`] = `
...
       },
-      {
-        fields: [
-          {
-            name: "metadatas",
-          },
-          {
-            name: "likes",
-          },
-          {
```
Comment (zifeo) [inline typegate/tests/simple/__snapshots__/class_syntax_test.ts.snap:62]: suspicious?

File: typegate/tests/simple/simple.py [hunk 7]
```diff
@@ -1,20 +1,23 @@
...
-            t.struct({"a": t.integer().named("arg1")}).named("inp"),
-            t.struct({"a": t.integer().named("deps")}).named("res"),
-            PredefinedFunMat("identity"),
+        rec=rec.with_policy(public),
+        test=deno.identity(
+            t.struct({"a": t.integer(name="arg1")}, name="inp"),
         )
-        .named("f")
-        .add_policy(policies.public()),
+        # .named("f")
```
Comment (zifeo) [inline typegate/tests/simple/simple.py:21]: rm?

File: typegate/tests/type_nodes/union_node_quantifier.py [hunk 8]
```diff
@@ -27,15 +30,17 @@
...
-        }
-    ).named("BasicPhone")
+        },
+        name="BasicPhone",
+    )
 
     phone = t.union([basic_phone, smartphone])
 
-    phone_register_materializer = ModuleMat("ts/union/phone_register.ts")
+    # phone_register_materializer = ModuleMat("ts/union/phone_register.ts")
```
Comment (zifeo) [inline typegate/tests/type_nodes/union_node_quantifier.py:39]: rm?

File: typegraph/python_next/typegraph_next/graph/params.py [hunk 9]
```diff
@@ -30,3 +58,162 @@ def hmac256(name: str) -> "core.Auth":
...
+@dataclass
+class OAuth2Params:
+    authorize_url: str
+    access_url: str
+    profile_url: Optional[str]
+    profiler: Optional[str]
+
+
+_oauth2_params = {
+    "digitalocean": OAuth2Params(
```
Comment (zifeo) [inline typegraph/python_next/typegraph_next/graph/params.py:97]: this could be set in the core?

**Review by zifeo — COMMENTED** [review]
Small thing to check and we are good :))

File: typegate/tests/auth/auth_test.ts (hunk 3 above)
Comment (Natoandro) [reply typegate/tests/auth/auth_test.ts:180]: It was a number...

Comment (Natoandro) [reply typegate/tests/auth/auth_test.ts:180]: It was a number...

**Pull request merged**